x-pack/plugins/endpoint/public/embeddables/resolver/view/defs.tsx

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo } from 'react';
+import { darken, saturate } from 'polished';
+
+import {
+  htmlIdGenerator,
+  // qualitative pallette functions.
+  //  See: https://elastic.github.io/eui/#/utilities/color-palettes
+  euiPaletteForTemperature,
+  euiPaletteForStatus,
+  colorPalette,
+} from '@elastic/eui';
+
+// Generating from `colorPalette` function: This could potentially
+//  pick up a palette shift and decouple from raw hex
+const [euiColorEmptyShade, , , , , euiColor85Shade, euiColorFullShade] = colorPalette(
+  ['#ffffff', '#000000'],
+  7
+);
+
+// A catalog of arbitraged colors
```

Comment:
![image](https://user-images.githubusercontent.com/35559/74969889-6a0edb80-53eb-11ea-82a5-38e40e63accf.png)
Can you explain your comment?

Reply:
I was using it generally to mean "moving an asset from one context to another for some kind of benefit", but point taken. I'll find a way to communicate that without using terms of art.